core/src/main/scala/akka/kafka/internal/DefaultProducerStage.scala

```diff
@@ -159,6 +156,9 @@ private class DefaultProducerStageLogic[K, V, P, IN <: Envelope[K, V, P], OUT <:
     }
   }
 
+  protected def emit(future: Future[OUT]): Unit =
```

Comment:
it's somewhat confusing to name this `emit` since that is a method in Akka's GraphStage. Can we come up with another name? `pushOut`?